src/schema/objects/entities.yaml

```
@@ -286,6 +276,15 @@ sample:
     The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
   type: string
   format: label
+seg:
+  name: seg
+  display_name: segmentation
```

The object key should be the long version of the name. `name` is what is seen in the entity `_<name>-<value>_`. And `display_name` is generally capitalized, imagining someone might write a GUI with a drop-down menu in it.

```suggestion
segmentation:
  name: seg
  display_name: Segmentation
```

---

Ah, thanks for correcting that. I didn't understand that.